test/ClientTest.php

```diff
@@ -524,4 +524,59 @@ public function testFormUrlEncodeSeparator()
         $rawRequest = $client->getLastRawRequest();
         $this->assertContains('foo=bar&baz=foo', $rawRequest);
     }
+
+    public function uriDataProvider()
+    {
+        return [
+            ['/example', true],
+            ['http://localhost/example', false]
+        ];
```

Comment:
Please add _names_ for each case in the provider. This makes it simpler to determine which case failed when a failure occurs. As an example:

```php
'valid-relative' => ['/example', true],
'invalid-absolute' => ['http://localhost/example', false],
```